_docs-sources/pipelines/maintain/drift-detection.md

```diff
@@ -0,0 +1,110 @@
+# Pipelines Drift Detection
+
+:::note
+Pipelines Drift Detection is only available to Pipelines Enterprise customers.
+:::
+
+## What Is Pipelines Drift Detection
+
+Infrastructure Drift occurs when the applied terragrunt cloud configuration differs from the commited Infrastructure as Code (IaC).
+
+Pipelines Drift Detection helps to mitigate Drift in your repositories by running `terragrunt plan` on infrastructure units. If the plan detects any units have drifted from their applied configuration Pipelines will open a Drift Detected Pull Request tracking this drift in your repository.
+
+When the Drift Detected Pull Request is merged, Pipelines will run `terragrunt apply` on all units where drift was detected.
+
+## Installing Pipelines Drift Detection
+
+Pipelines Drift Detection is automatically vended in new Enterprise repositories.
+
+To add Drift Detection to an existing repository:
```

Comment:
Hrm.  If our stance is that enterprise customers get it automatically, who is the audience for the manual instructions?

Reply:
Enterprise customers that are upgrading, but maybe this should live somewhere else then?

Reply:
Removed - we can put this in release notes instead.